stubs/CoreGenericFunctions.phpstub

```diff
@@ -484,6 +484,27 @@ function strtolower(string $str) : string {}
  */
 function strtoupper(string $str) : string {}
 
+/**
+ * @psalm-pure
+ *
+ * @psalm-flow ($str, $to) -> return
+ */
+function strtr(string $str, string $from, string $to) : string {}
+
+/**
+ * @psalm-pure
+ *
+ * @psalm-flow ($str, $replace_pairs) -> return
+ */
+function strtr(string $str, array $replace_pairs) : string {}
```

Comment:
Psalm doesn't handle duplicate stubbed signatures for the same function, unfortunately. This will need to be a plugin instead (similar plugins are here: https://github.com/vimeo/psalm/tree/master/src/Psalm/Internal/Provider/ReturnTypeProvider)